pkg/api/org_test.go

```diff
@@ -194,7 +194,7 @@ func TestAPIEndpoint_PutCurrentOrgAddress_AccessControl(t *testing.T) {
 // `/api/orgs/` endpoints test
 
 // setupOrgsDBForAccessControlTests creates orgs up until orgID and fake user as member of org
-func setupOrgsDBForAccessControlTests(t *testing.T, db sqlstore.Store, c accessControlScenarioContext, orgID int64) {
+func setupOrgsDBForAccessControlTests(t *testing.T, db *sqlstore.SQLStore, c accessControlScenarioContext, orgID int64) {
```

Comment:
The reason why we add the fakes is to not use real db in api tests. This is an inbetween solution. I think we should discuss our testing strategy in more detail. I really would not like to add more of the orgimpl in our api tests.

Reply:
I can second @idafurjes response here. The test fixtures are complicated and look ugly, but (so far) this pattern has been the most ergonomic. There are frequently a lot of services / mocks / fakes used in tests that all use the same "base" `sqlstore.SQLStore`.

Reply:
in the specific case we don't need a fake because we actually need [to create an org](https://github.com/grafana/grafana/blob/11b997170c13802102100774c3021a4bd957c813/pkg/api/org_test.go#L206. Also - my mistake - this has nothing to do with the handler but my assumption is that eventually we want to remove `sqlstore.CreateOrgWithMember` and always call the service. So it my assumption is true, this is a good opportunity to change here.

Reply:
we should still discuss this further